build.sbt

```diff
@@ -15,27 +15,27 @@ lazy val coreTest =
   Project(id = "akka-projection-core-test", base = file("akka-projection-core-test"))
     .configs(IntegrationTest)
     .settings(Defaults.itSettings)
-    .settings(Protobuf.settings)
+    .settings(Dependencies.coreTest)
     .settings(publish / skip := true)
-    .dependsOn(core % "compile->compile;test->test")
-    .dependsOn(testkit % "test->test")
+    .dependsOn(core)
+    .dependsOn(testkit % "test")
```

Comment:
`testkit` is a library. In core-test we use it in test scope. We were doing it transitively vie `testkit` tests scope, that's not correct.